saltedge-ios-swift/Classes/API/SEConnectionFetcher.swift

```diff
@@ -34,6 +34,7 @@ class SEConnectionFetcher {
         HTTPService<SEConnection>.makeRequest(ConnectionRouter.create(params)) { response in
             switch response {
             case .success(let value):
+                completion?(.success(value))
```

Comment:
Also, please add `completion` block in all other methods (`updateConnectionStatus`, `reconnectConnection`, `refreshConnection`, `interactiveConnection`).